mod-kb-ebsco-java/src/main/resources/domain/mod-kb-ebsco-java/features/tags.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Tags
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers = { 'Content-Type': 'application/vnd.api+json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/vnd.api+json' }
+
+  @Undefined
+  Scenario: GET all tags with 200 on success
+    * print 'undefined'
+
+  @Undefined
+  Scenario: GET all tags filtered by record types with 200 on success
+    * print 'undefined'
+
+  @Undefined
+  Scenario: GET all tags filtered by record types should return 400 if filter parameter is invalid
+    * print 'undefined'
+
+  @Undefined
+  Scenario: GET all unique tags with 200 on success
+    * print 'undefined'
+
+  @Undefined
+  Scenario: GET all unique tags filtered by record types with 200 on success
+    * print 'undefined'
+
+  @Undefined
+  Scenario: GET all unique tags filtered by record types should return 400 if filter parameter is invalid
```

Comment:
minor: please separate positive and negative cases with comments for better visibility

and we have only one negative case?

Reply:
Am I need to separate positive and negative cases in whole features that I created? 
Yes, In the [**documentation**](https://s3.amazonaws.com/foliodocs/api/mod-kb-ebsco-java/tags.html#), just one negative case